bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -660,6 +660,215 @@ namespace JsRTApiTest
         JsRTApiTest::RunWithAttributes(JsRTApiTest::ExternalFunctionTest);
     }
 
+    JsValueRef CALLBACK ExternaEnhancedFunctionTestCallback(JsValueRef callee, JsValueRef *arguments, unsigned short argumentCount, JsNativeFunctionInfo *info, void *callbackData)
```

Comment:
Typo: should be `ExternalEnhancedFunctionTestCallback`

Reply:
Thanks for that, good catch.